group.go

```diff
@@ -30,11 +32,11 @@ type Group struct {
 	// DescendantRepositories RepositoryConnection `json:"descendantRepositories,omitempty"`
 	// DescendantServices ServiceConnection `json:"descendantServices,omitempty"`
 	// DescendantSubgroups    SubgroupConnection   `json:"descendantSubgroups,omitempty"`
-	// DescendantTeams        TeamConnection       `json:"descendantTeams,omitempty"`
+	// DescendantTeamsConnection Connection `json:"descendantTeams,omitempty"`
```

Comment:
Was having a hard time with these since it was adding a query parameter that doesn't exist, e.g. `descendantTeamsConnection{}`.  Also, I'm not fully understanding why these would need to be in the struct so might need a quick sync when @rocktavious is back.